csharp/AppTests/TestClass.cs

```diff
@@ -0,0 +1,25 @@
+using NUnit.Framework;
+using Interview;
+
+namespace Interview
+{
+    [TestFixture]
+    public class TestClass
+    {
+        private Class classInstance;
+
+        [SetUp]
+        public void SetUp()
+        {
+            classInstance = new Class();
+        }

```

Comment:
I would remove the opinionated approach of creating the sut in the setup, and storing as a private field. Let the candidates refactor to that if they think fit.  

Perhaps just basically some empty shells showing the types of testing methods

```suggestion

        [SetUp]
        public void SetUp()
        {
            // do some setup before each test
        }
        
        [Test]
        public void TestMethod()
        {
            Assert.IsTrue(false);
        }```